cypress/e2e/singleMovie.cy.js

```diff
@@ -0,0 +1,42 @@
+describe("navigate to individual movie", () => {
+  beforeEach(() => {
+    cy.intercept(
+      "GET",
+      "https://rancid-tomatillos.herokuapp.com/api/v2/movies",
+      {
+        statusCode: 200,
+        fixture: "allMovies.json",
+      }
+    )
+    .visit("http://localhost:3000/");
+  });
+
+  it("should show user landing page with all movies, titles, and release dates", () => {
+    cy.intercept(
+      "GET",
+      "https://rancid-tomatillos.herokuapp.com/api/v2/movies/760104",
+      {
+        statusCode: 200,
+        fixture: "singleMovie.json",
+      }
+    )
+    cy.get(".card-container").find(".card").should("have.length", 4);
+    cy.get(".card").last().click()
+    cy.contains('p', 'In 1979')
+    cy.contains('p', '2022')
+    cy.contains('p', 'Runtime:')
+    cy.contains('button', 'Go Back Home').click()
+  });
+
+  it("should display the landing page to the user", () => {
```

Comment:
We don't need to stub the fetch for movies a second time because we already added it in the beforeEach hook. Additionally, the first test ends with a click. If we get rid of lines 29-39 we can combine these two it blocks and resolve both of these issues!